components/brave_mobile_subscription/renderer/android/subscription_render_frame_observer.h

```diff
@@ -54,12 +54,14 @@ class SubscriptionRenderFrameObserver : public content::RenderFrameObserver {
   FRIEND_TEST_ALL_PREFIXES(SubscriptionRenderFrameObserverTest, ExtractParam);
   FRIEND_TEST_ALL_PREFIXES(SubscriptionRenderFrameObserverTest, IsValueAllowed);
   enum class Product { kVPN = 0, kLeo = 1 };
+  enum class Page { kInitialLandingPage = 0, kResultLandingPage = 1 };
 
   bool EnsureConnected();
   void OnGetPurchaseToken(const std::string& purchase_token);
   void OnGetPurchaseTokenOrderId(const std::string& purchase_token,
                                  const std::string& order_id);
   std::string ExtractParam(const GURL& url, const std::string& name) const;
+  std::string ExtractPath(const GURL& url) const;
```

Comment:
frankly speaking, this is not a method, just a one liner util function, so it's better to move it outside of the class, just to an anonimous namespace (together with `ExtractParam` and `IsValueAllowed`